package.json

```diff
@@ -48,7 +48,7 @@
   "dependencies": {
     "@code-pushup/portal-client": "^0.6.1",
     "@isaacs/cliui": "^8.0.2",
-    "@poppinss/cliui": "^6.3.0",
+    "@poppinss/cliui": "^6.4.0",
```

Comment:
This is needed to get the latest features from https://github.com/poppinss/cliui/pull/16 in